### ISSUE TITLE
Axes from mods and copper axe not mining trees

### ISSUE THREAD

**Issue by optionalduck** [nice-john/BetterBeginnings_1.0_Forge1.20.1#2]
I have a customized iron axe from tetra as well as a copper axe which both dont work on logs or trees from dynamic trees, however the stone hatchet and vanilla iron axe do work

**Comment by nice-john** [comment]
uhhhh not sure why this would happen bc the copper axe is an "AxeItem" so it should work fine. probably a dynamic trees issue if the problem is happening with tetra as well

**Comment by optionalduck** [comment]
it happens on vanilla logs also and gives the message from this mod when you break them so that's why I think its from this mod

**Comment by nice-john** [comment]
I think I found the error im fixing it rn. I will push out the fix tonight or tomorrow morning

**Comment by nice-john** [comment]
Should be fixed now. Download the newest version it should finish processing shortly and let me know if it's solved

**Comment by optionalduck** [comment]
Looks like all the axes work but the tetra ones. I don't know much about this stuff but I would assume it is because it according to the game isn't an axe but a two handed tool with an axe head on it. IDK how to read the nbt tags to see if it has an axe tag or not but maybe that is the problem.

**Comment by nice-john** [comment]
Yeah so it turned out I had it set up using the "axe" tag actually rather than checking if it was an 'AxeItem'. This fixed the copper axe bc I never added any tags to my items, and it should make the mod more compatible with others since I'm sure lots of other mod authors have neglected to tag their items too. I figured that was the issue with tetra too. But I guess your tetra axe is neither. I'll look into this a little more I might be able to figure something out tomorrow but I'm a little stumped and this might have to be fixed on the tetra side of things but I have no idea how that mod works I've never played with it

**Comment by optionalduck** [comment]
Yeah it works with vanilla style axes from other mods in my modpack so it seems like it is something with the way tetra's tools work.  I will report the issue to them tomorrow once I get the time.
